automation/test-execution/ansible/tests/integration/helpers/container_helper.py

```
@@ -0,0 +1,205 @@
+    Stop a container.
+
+    Args:
+        runtime: Container runtime
+        container_id: Container ID or name
+        timeout: Stop timeout in seconds
+
+    Returns:
+        True if stopped successfully
+    """
+    try:
+        subprocess.run(
```

Ruff (0.15.12)

[error] `subprocess` call: check for execution of untrusted input

(S603)